redux/features/pokemon/slice.js

```diff
@@ -8,15 +8,15 @@ import {
 const pokemonSlice = createSlice({
   name: SLICE_NAME,
   initialState: {
-    list: [],
+    pokemonList: [],
     hasMore: true,
-    isLoading: true,
+    isLoading: false,
```

Comment:
Use a different isLoading variable for the pokemonList and pokemonData. You can name them `isLoadingPokemonList` and `isLoadingPokemonData`.